packages/caip/src/adapters/coingecko/utils.ts

```diff
@@ -1,13 +1,17 @@
+/* eslint-disable no-empty */
```

Comment:
can we remove this?

Reply:
yeah adding comment in the catch to explain and reduce need for the disable